visualization/src/point_picking_event.cpp

```diff
@@ -189,12 +191,12 @@ pcl::visualization::PointPickingCallback::performSinglePick (
 /////////////////////////////////////////////////////////////////////////////////////////////
 int
 pcl::visualization::PointPickingCallback::performAreaPick (vtkRenderWindowInteractor *iren,
-                                                           std::vector<std::vector<int> > &indices)
+                                                           std::vector<std::vector<int> > &indices, vtkActorCollection *actors )
```

Comment:
Why not use `std::map<std::string, std::vector<int>>` directly? We do have the name here as well (`actor.first`), right?

Reply:
just to be clear, pass map of actors with no points the method and inside method if find actor emplace indices? So in map would be only displayed cloud actors(names)?

Reply:
Yeah. Pass in the `std::map` (or perhaps return it instead assuming a blank map is an error) and move it into the ctor.

> inside method if find actor emplace indices?

> So in map would be only displayed cloud actors(names)? that would remove uninitialized name I guess

I'm sorry, I didn't get these.